client/components/SearchViews/CustomerQandA.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Header = styled.h1`
+  font-size: 14px;
+  font-weight: 400;
+  color: #565959!important;
+`;
+
+const LinkWrapper = styled.div`
+  margin-top: 16px;
+`;
+
+const Link = styled.a`
+  color: #007185;
+  text-decoration: none;
+  cursor: pointer;
+  &:hover { text-decoration: underline; }
+`;
+
+const CustomerQandA = () => {
+  return (
+    <>
```

Comment:
Can this " <> " be taken out?

Reply:
No, as adjacent jsx elements must be wrapped in an enclosing tag. I am using <> and </> as [react fragments ](https://reactjs.org/docs/fragments.html#short-syntax) to avoid adding extra nodes/divs to the DOM.

Reply:
Gotcha!